xsd/netex-nl-basic.xsd

```diff
@@ -265,17 +262,17 @@ De TypeOfFrame standaardenumeraties hebben een expliciete 'version', nl. de vers
 	</xsd:complexType>
 -->
 	<xsd:complexType name="VersionOfObjectRefStructure">
-		<xsd:attribute name="version" type="VersionIdType" use="optional" default="any" form="unqualified"/>
+		<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
 		<xsd:attribute name="ref" type="ObjectIdType" use="required" form="unqualified"/>
 	</xsd:complexType>
 	<xsd:complexType name="VersionOfObjectRefStructure-with-class">
 		<xsd:attribute name="nameOfRefClass" type="NameOfClass" use="required" form="unqualified"/>
-		<xsd:attribute name="version" type="VersionIdType" use="optional" default="any" form="unqualified"/>
+		<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
 		<xsd:attribute name="ref" type="ObjectIdType" use="required" form="unqualified"/>
 	</xsd:complexType>
 	<xsd:complexType name="VersionOfObjectRefStructure-with-optional-class">
 		<xsd:attribute name="nameOfRefClass" type="NameOfClass" use="optional" form="unqualified"/>
-		<xsd:attribute name="version" type="VersionIdType" use="optional" default="any" form="unqualified"/>
+		<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
```

Comment:
Bewuste keuze default hier weg te halen?

Reply:
ja: als het attribuut required is, overschrijf je altijd de default waarde, dus is die niet nodig